### PR TITLE
Add DB backend repos to repo list

### PR DESCRIPTION
**Pull request by chilts**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ Most repositories are [available via GitHub](https://github.com/mozilla?utf8=%E2
 - [fxa-auth-server](https://github.com/mozilla/fxa-auth-server)
 - [fxa-oauth-server](https://github.com/mozilla/fxa-oauth-server)
 - [fxa-profile-server](https://github.com/mozilla/fxa-profile-server)
+- [fxa-auth-db-server](https://github.com/mozilla/fxa-auth-db-server)
+- [fxa-auth-db-mem](https://github.com/mozilla/fxa-auth-db-mem)
+- [fxa-auth-db-mysql](https://github.com/mozilla/fxa-auth-db-mysql)
```
Comment (pdehaan) [inline docs/index.md]: fxa-customs-server?

Comment (chilts) [inline docs/index.md]: Oh fine!

Comment (pdehaan) [inline docs/index.md]: @chilts: Hey, don't get mad at me. I'm just going through FxA looking repos from https://github.com/mozilla/?utf8=%E2%9C%93&query=fxa-

Comment (chilts) [inline docs/index.md]: I'm not mad, just a little eccentric.

**Comment by chilts** [comment]
Thanks @vladikoff.

**Comment by vladikoff** [comment]
Yesss!

**Pull request merged**